### ISSUE TITLE
pytorch lightning version question

### ISSUE THREAD

**Issue by Value-Jack** [naver-ai/facetts#6]
furthermore, i have the pl version problem.

  File "/home/zjli/anaconda3/envs/facetts/lib/python3.8/site-packages/pytorch_lightning/strategies/ddp.py", line 179, in setup
    self.setup_optimizers(trainer)
  File "/home/zjli/anaconda3/envs/facetts/lib/python3.8/site-packages/pytorch_lightning/strategies/strategy.py", line 128, in setup_optimizers
    self.optimizers, self.lr_scheduler_configs, self.optimizer_frequencies = _init_optimizers_and_lr_schedulers(
  File "/home/zjli/anaconda3/envs/facetts/lib/python3.8/site-packages/pytorch_lightning/core/optimizer.py", line 195, in _init_optimizers_and_lr_schedulers
    _validate_scheduler_api(lr_scheduler_configs, model)
  File "/home/zjli/anaconda3/envs/facetts/lib/python3.8/site-packages/pytorch_lightning/core/optimizer.py", line 350, in _validate_scheduler_api
    raise MisconfigurationException(
pytorch_lightning.utilities.exceptions.MisconfigurationException: The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.

**Comment by Value-Jack** [comment]
The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.

**Comment by lee-jiyoung** [comment]
pytorch-lightning==1.5.8
